doc/manual/manual.tex

```diff
@@ -1595,14 +1741,198 @@ \subsubsection{Compressible models}
 where $\frac{1}{\rho} \frac{\partial \rho}{\partial p}$ is often referred to
 as the compressibility.
 
-In the implementation used in \aspect{}, this equation replaces
+For this approximation, this equation replaces
 \eqref{eq:stokes-2}. It has the advantage that it retains the symmetry of the
 Stokes equations if we can treat the right hand side of
 \eqref{eq:stokes-2-compressible} as known. We do so by evaluating $\rho$ and
 $\mathbf u$ using the solution from the last time step (or values extrapolated
-from previous time steps).
+from previous time steps), or using a nonlinear solver scheme.
+
+\note{This is the default approximation \aspect{} uses to model compressible convection,
+see Section~\ref{sec:combind_formulations}.}
+
+
+\newpage
+\subsection{Choosing a formulation in \aspect{}}
+
+After discussing different reasonable approximations for modelling compressible or 
+incompressible mantle convection, we will now describe the different steps one has 
+to take to use one of these approximations in a computation.  
+This includes 
+\begin{enumerate}
+\item Choosing an approximation for the mass conservation equation, 
+\item Choosing an approximation for the density in the energy balance, and deciding which heating terms should be included, 
+\item Formulating the buoyancy term in the material model to be used on the right-hand side of the momentum equation, 
+\item Prescribing a suitable reference state for the temperature, pressure, and density, the adiabatic profile.
```

Comment:
do you mean "...and density; i.e., the adiabatic profile"?

Maybe also add "..., if necessary for the approximations chosen in the first three steps".